### PR TITLE
[WIP] generate yaml by helm chart template

### PR DESCRIPTION
**Pull request by MaazMS**
Generate yaml by helm chart template

### CODE REVIEW

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-aef1b21d`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-dc8b68c3`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-10bd402c`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-a9f03dac`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-32abe751`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-fb96e4a7`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-6d54dbe1`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-ad5f25e8`

**Comment by stakater-user** [comment]
@MaazMS Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-12-980ff19e`

File: charts/workshop-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -33,69 +35,25 @@ spec:
         - --upstream=http://127.0.0.1:8080/
         - --logtostderr=true
         - --v=10
-        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.7.0
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
```
Comment (rasheedamir) [inline charts/workshop-operator/templates/deployment.yaml:37]: @MaazMS i want to use this one

File: charts/workshop-operator/templates/deployment.yaml [hunk 2]
```diff
@@ -33,69 +35,25 @@ spec:
...
-          value: {{ .Values.watchNamespaces | join "," | quote }}
-        - name: CONFIG_SECRET_NAME
-          value: "{{ default "workshop-secret" .Values.configSecretName }}"
-        - name: ENABLE_WEBHOOKS
-          value: "{{ default true .Values.webhook.enabled }}"
-        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
-        securityContext:
-          allowPrivilegeEscalation: false
+        image: quay.io/stakater/workshop-operator:1.2
```
Comment (rasheedamir) [inline charts/workshop-operator/templates/deployment.yaml:48]: @MaazMS we don't push to quay! we push docker.io!
https://hub.docker.com/r/stakater/workshop-operator/tags?page=1&ordering=last_updated

Comment (MaazMS) [reply charts/workshop-operator/templates/deployment.yaml:48]: @rasheedamir is this correct image url for docker `image: docker.io/stakater/workshop-operator`

**Pull request closed without merging**